fitbenchmarking/controllers/base_controller.py

```diff
@@ -58,6 +58,26 @@ class Controller:
                        'steepest_descent': [],
                        'global_optimization': []}
 
+    #: Within the controller class, you must define the list
+    #: ``jacobian_enabled_solvers`` if any of the minimizers
+    #: for the specific software are able to use jacobian
+    #: information.
+
```

Comment:
The docs only show the second paragraph here. Is that intentional? If we want the full thing we need to have a single block where each line starts with `#:` (including any blank lines).

Reply:
Ah, thanks for spotting this - have now updated it so that both paragraphs are shown in the docs